charts/azuremonitor-containers/values.yaml

```diff
@@ -78,6 +78,9 @@ amalogs:
     clusterRegion: <your_cluster_region>
   rbac: true
   sidecarscraping: true
+  syslog:
```

Comment:
We should add a note that something like, "syslog collection requires additional config dependencies on the node and its not supported and please open an service ticket  if there is an syslog collection requirement"